### PR TITLE
Chris

### PR DESCRIPTION
**Pull request by cplee1**
Misc commits

- Fixed error bars in Sieber data, which were set as the errors in pulse energies not flux densities
- Added 2 papers to `bad' paper list because I noticed duplicates
- Added Zhao to paper list in docs
- Changed some limits/tolerance to help plots find the best fits

### CODE REVIEW

File: pulsar_spectra/catalogue_papers/Sieber_1973_raw_to_yaml.py [hunk 1]
```diff
@@ -46,8 +46,9 @@
             fact_err = 0.5
         elif "c" in row[4]:
             fact_err = 0.75
-        pulsar_dict[pulsar]["Flux Density error mJy"] += [round(flux*fact_err, 3)]
+
+        pulsar_dict[pulsar]["Flux Density error mJy"] += [round(flux*fact_err/query['P0'][pid], 3)]
 
 with open("Sieber_1973.yaml", "w") as cat_file:
-    cat_file.write(json.dumps(pulsar_dict))
+    cat_file.write(json.dumps(pulsar_dict, indent=4))
```
Comment (NickSwainston) [inline pulsar_spectra/catalogue_papers/Sieber_1973_raw_to_yaml.py:53]: It's good that you've remembered to do this but I think we should use indent=1. Just so the file is smaller. We should eventually do this for all of the yamls.

File: pulsar_spectra/models.py [hunk 2]
```diff
@@ -202,7 +202,7 @@ def model_settings(print_models=False):
         ],
         "high_frequency_cut_off_power_law" : [
             high_frequency_cut_off_power_law,
-            "pl high cut-off",
+            "pl hard cut-off",
```
Comment (NickSwainston) [inline pulsar_spectra/models.py:205]: I don't think I like hard cut off, is there a reason for this change?

File: pulsar_spectra/spectral_fit.py [hunk 3]
```diff
@@ -154,7 +154,8 @@ def plot_fit(freqs_MHz, fluxs_mJy, flux_errs_mJy, ref, model, iminuit_result, fi
             freqs_ref = np.array(data_dict[ref]['Frequency MHz'])
             fluxs_ref = np.array(data_dict[ref]['Flux Density mJy'])
             flux_errs_ref = np.array(data_dict[ref]['Flux Density error mJy'])
-            (_, caps, _) = ax.errorbar(freqs_ref, fluxs_ref, yerr=flux_errs_ref, linestyle='None', mec='k', markeredgewidth=marker_border_thickness, elinewidth=errorbar_linewidth, capsize=capsize, label=ref.replace('_',' '))
+            (_, caps, _) = ax.errorbar(freqs_ref, fluxs_ref, yerr=flux_errs_ref, linestyle='None', mec='k', markeredgewidth=marker_border_thickness, \
```
Comment (NickSwainston) [inline pulsar_spectra/spectral_fit.py:157]: Pretty sure you don't need a \. Also, I prefer the format:
```
(_, caps, _) = ax.errorbar(
    freqs_ref, 
    fluxs_ref,
    yerr=flux_errs_ref,
    linestyle='None',
    mec='k',
    markeredgewidth=marker_border_thickness,
    elinewidth=errorbar_linewidth,
    capsize=capsize,
    label=ref.replace('_',' '),
)
```
It's a bit easier to read and makes adding more options/arguments easier

File: pulsar_spectra/catalogue.py [hunk 4]
```diff
@@ -273,7 +273,7 @@ def collect_catalogue_fluxes(only_use=None, exclude=None, query=None):
 
     # Add the antf to the cataogues
     antf_dict = all_flux_from_atnf(query=query)
-    antf_incorrect_refs = ["Zhao_2019", "Mignani_2017", "Stovall_2015"]
+    antf_incorrect_refs = ["Zhao_2019", "Mignani_2017", "Stovall_2015", "Bell_2016", "Lorimer_1995"]
```
Comment (NickSwainston) [inline pulsar_spectra/catalogue.py:276]: Can you please send me a pulsar or two that you noticed these errors. I'll have a look into them and decide if we have to notify ATNF or edit our code.

**Review by NickSwainston — CHANGES_REQUESTED** [review]
Another nice pull request. I just have a few minor comments.

File: pulsar_spectra/catalogue.py (hunk 4 above)
Comment (cplee1) [reply pulsar_spectra/catalogue.py:276]: dups_1
dups_5
dups_4
dups_3
dups_2

Comment (NickSwainston) [reply pulsar_spectra/catalogue.py:276]: I've checked and 

Lorimer_1995 ATNF data is correct but were scaled to by their spectral index for the ATNF frequencies

Bell_2016 ANTF data flux uncertainties are off by an order of magnitude

**Review by NickSwainston — APPROVED** [review]

**Pull request merged**